### PR TITLE
IDVA6-1251: Add confirmation page for company removal

### PR DESCRIPTION
**Pull request by tferns-ch**
Before this endpoint is re-directed to, we need to set some data in the session, then we pop it when we render this page.
```typescript
import { setExtraData } from "...";
setExtraData(req.session, constants.LAST_REMOVED_COMPANY_NAME, "Name");
setExtraData(req.session, constants.LAST_REMOVED_COMPANY_NUMBER, "Number");
res.redirect(constants.REMOVE_COMPANY_CONFIRMED_FULL_URL);
```

## Changes:
- Create new view 'confirmation-company-removed.njk'
- Add route for company removal confirmation
- Implement RemoveCompanyConfirmedHandler
- Update router and constants for new feature
- Add utility function to pop extra data from session
- Internationalize text content (add English and Welsh translations)

## Screenshot:
Screenshot 2024-09-18 at 15 15 03

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/routers/handlers/yourCompanies/removeCompanyConfirmedHandler.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import { getTranslationsForView } from "../../../lib/utils/translations";
+import * as constants from "../../../constants";
+import { popExtraData } from "../../../lib/utils/sessionUtils";
+import { ViewData } from "../../../types/util-types";
+
+export class RemoveCompanyConfirmedHandler extends GenericHandler {
+    async execute (req: Request): Promise<ViewData> {
+        this.viewData = await this.getViewData(req);
+        this.viewData.lang = getTranslationsForView(
+            (req as any).lang,
```
[the remark this answers is not in the document]
Comment (tferns-ch) [reply src/routers/handlers/yourCompanies/removeCompanyConfirmedHandler.ts:12]: Where was this type added? do I need to rebase on main?

[the remark this answers is not in the document]
Comment (tferns-ch) [reply src/routers/handlers/yourCompanies/removeCompanyConfirmedHandler.ts:12]: Ah it was added in https://github.com/companieshouse/your-companies-web/pull/186/files

[the remark this answers is not in the document]
Comment (tferns-ch) [reply src/routers/handlers/yourCompanies/removeCompanyConfirmedHandler.ts]: I've fixed in https://github.com/companieshouse/your-companies-web/pull/185/commits/d171a4d9c082f03e2e51699d2b7c9f133ac737dd after rebasing on main

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request closed without merging**